test/Jamfile.v2

```diff
@@ -144,6 +130,35 @@ test-suite graph_test :
     [ run delete_edge.cpp ]
     ;
 
+alias graph_test_no_msvc14 : :
+        <toolset>msvc
+        <toolset-msvc:version>14.0
+        <address-model>64
+    ;
+
+alias graph_test_no_msvc14 :
+    [ run astar_search_test.cpp ]
+    [ run basic_planarity_test.cpp ]
+    [ run make_bicon_planar_test.cpp ]
+    [ run make_maximal_planar_test.cpp ]
+    [ run all_planar_input_files_test.cpp
+        ../../filesystem/build
+        ../../system/build
+        : $(PLANAR_INPUT_FILES) ]
+    [ run parallel_edges_loops_test.cpp
+        ../../filesystem/build
+        ../../system/build
+        : $(PLANAR_INPUT_FILES) ]
+    [ run random_spanning_tree_test.cpp ../build//boost_graph ]
+    [ run mas_test.cpp ../../test/build//boost_unit_test_framework/<link>static : $(TEST_DIR) ]
+    [ run stoer_wagner_test.cpp ../../test/build//boost_unit_test_framework/<link>static : $(TEST_DIR) ]
+    ;
+
+test-suite graph_test :
+    graph_test_regular
+    graph_test_no_msvc14
+    ;
+
```

Comment:
Why all these other changes that seem unrelated to `minimum_degree_ordering`? Maybe better to put them in a different PR?

Reply:
Oh, I get it now, you merged your parameter fix in here too. Looks like we should get that one merged asap.